src/components/Api.js

```diff
@@ -1,2 +1,5 @@
 export const API_URL = "https://blog-api344.herokuapp.com/blogposts";
 
+export const REGISTER_API = "https://blog-perm.herokuapp.com/api/register"
+
+export const LOGIN_API = "https://blog-perm.herokuapp.com/api-auth/login/"
```

Comment:
Are you using three different api services?

Reply:
yes sir